services/storage/app/models/concerns/has_attachment.rb

```diff
@@ -58,6 +58,7 @@ def feature_set; Settings.feature_set end
 
     def object_scope; 'tenants' end
 
+    # TODO: if we need the tenant in a different format, that should be the responsibility of the Tenant model
```

Comment:
Agree but the current issue is that `Apartment::Tenant.current` returns a string straight from the adapter: https://github.com/influitive/apartment/blob/development/lib/apartment/adapters/postgresql_adapter.rb#L68